custom_components/rivian/const.py

```diff
@@ -179,6 +192,12 @@
             name="Gear Selector",
             icon="mdi:car-shift-pattern",
             key=f"{DOMAIN}_dynamics_propulsion_status_prndl",
+            device_class=SensorDeviceClass.ENUM,
+            options=[
+                "Drive",
+                "Park",
+                "Reverse",
```

Comment:
`Neutral` though not a gear I use frequently, I would imagine this gets pushed through as `neutral`.
We should check that and verify with an actual truck/suv.

Reply:
Before adding the `Neutral` option, the state change to neutral was ignored and the Gear Selector sensor retained its prior state.  I've added `Neutral` to the list of possible state values, and confirmed that it is working locally.  (The native value is `neutral`, but the options list takes into account the capitalized value from the lambda function.)

![IMG_1907](https://user-images.githubusercontent.com/18428022/218507365-712d18e2-0212-48eb-aa51-5861b7955a3a.jpg)